packages/table/src/common/grid.ts

```diff
@@ -5,6 +5,8 @@
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
 
+import * as React from "react";
```

Comment:
needs this now, because one of the methods here export a react interface through `Object.assign`...